scripts/extension-version.js

```diff
@@ -0,0 +1,81 @@
+const semVer = require('semver')
```

Comment:
Why are you not importing semver with `import { expect } from 'semver'`?

Reply:
same goes for `chai`

Reply:
Because this is a JS file (no TS, no bundler) and vanilla JS doesn't support the import syntax (I think in node 14 + .mjs files it would work). 

This syntax will work with all node versions.